app/docs/enterprise/0.32-x/installation/ubuntu.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to install Kong Enterprise and PostgreSQL onto Ubuntu
+---
+
+## Install Kong Enterprise
+
+curl the EE binary for Ubuntu (or Debian) from Bintray to your Ubuntu server
+
+Then, with the file local:
+
+```bash
+$ sudo apt-get update
+$ sudo apt-get install openssl libpcre3 procps perl
+$ sudo dpkg -i kong-enterprise-edition-0.32.zesty.all.deb
+```
+NOTE: EE file may differ in last step above. In addition to Zesty, we currently build for Precise, Trusty, and Xenial.
+
+## Install PostgreSQL
+```bash
+$ sudo apt-get install postgresql
+$ sudo -i -u postgres (puts you into new shell)
+```
+
+Create `kong` user
+
+```bash
+$ psql
+> CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong'; 
+> \q
+$ exit
+```
+
+## Finish Kong Enterprise setup steps
+
+```bash
+# Add contents of your license file (copy & paste)
+$ sudo vi /etc/kong/license.json
+$ sudo cp /etc/kong/kong.conf.default /etc/kong/kong.conf
+# Uncomment and add 'kong' to pg_password line
+$ sudo vi /etc/kong/kong.conf
+```
+
+### Run migrations and start kong
+```bash
+$ sudo /usr/local/bin/kong migrations up
```

Comment:
probably should add this for 0.31 as well?

Reply:
hmm, you mean like "Ubuntu" here: https://getkong.org/docs/enterprise/0.31-x/ which links to here: https://getkong.org/docs/enterprise/0.31-x/installation/ubuntu/

So, already in 0.31?

Reply:
It looks like this command changed, so I'm commenting that 0.31 should include this change as well?

`kong migrations up` -> `sudo /usr/local/bin/kong migrations up`

Reply:
Thank you eagle eyes!